plasma_cash/child_chain/child_chain.py

```diff
@@ -68,3 +68,9 @@ def apply_transaction(self, transaction):
 
     def get_current_block(self):
         return rlp.encode(self.current_block).hex()
+
+    def get_block(self, blknum):
+        return rlp.encode(self.blocks[int(blknum)]).hex()
+
+    def get_tx_proof(self, blknum, uid):
+        return self.blocks[int(blknum)].merkle.create_merkle_proof(int(uid))
```

Comment:
I think it's better to just assume getting int as input? the int conversion is better to be done on the side whoever calls this.